src/libs/dutil/pathutil.cpp

```diff
@@ -14,6 +14,7 @@
 #include "precomp.h"
 
 #define PATH_GOOD_ENOUGH 64
+static const DWORD TEMP_FILE_CREATE_RETRY_COUNT = 3;
```

Comment:
This retry limit is too small given how this function is used. I'd do at least, 300, which would be 30 second wait.